apps/condo/domains/notification/schema/Device.js

```diff
@@ -71,7 +98,16 @@ const Device = new GQLListSchema('Device', {
             type: Json,
             isRequired: false,
         },
-
+    },
+    kmigratorOptions: {
+        constraints: [
+            {
+                type: 'models.UniqueConstraint',
+                fields: ['deviceId', 'appId'],
```

Comment:
It’s redundant. The device is already unique

Reply:
it's for backward compatibility, deviceId constraint will be removed later

Reply:
removed deviceId unique constraint